config/initializers/menus.rb

```diff
@@ -32,7 +32,14 @@
 Redmine::MenuManager.map :top_menu do |menu|
   # projects menu will be added by
   # Redmine::MenuManager::TopMenuHelper#render_projects_top_menu_node
-
+  menu.push :projects,
+            { controller: '/projects', project_id: nil, action: 'index' },
+            context: :modules,
+            caption: I18n.t('label_projects_menu'),
+            if: Proc.new {
+              (User.current.logged? || !Setting.login_required?) &&
+                User.current.allowed_to?(:all_projects, nil, global: true)
```

Comment:
This permission does not exist either. It leads to the same result as before, showing the link only for admins but for nobody else.